powerpacks.go

```diff
@@ -50,15 +50,31 @@ type ShortCode struct {
 	Added_on         string `json:"added_on,omitempty"`
 }
 
+type Tollfree struct {
+	Number_pool_uuid string `json:"number_pool_uuid,omitempty"`
```

Comment:
under_score variable names are valid, they are not idiomatic. Follow the standard

Reply:
Existing structs for longcodes and shortcode uses number_pool_uuid in their structs so should i leave it like this only to maintain uniformity or should i remove the underscore